charts/px/README.md

```diff
@@ -134,9 +134,39 @@ Incorrect ETCD URL provided. It is either not reachable or is incorrect...
 
 ```
 Ensure the correct etcd URL is set as a parameter to the `helm install` command. 
-The hook resource wouldnt be cleared by itself and hence please delete the `pre-install-hook` after identifying the root cause of the failure. 
 ```
-kubectl delete jobs px-etcd-preinstall-hook -nkube-system
+
+```
+Follow the steps mentioned in the pre-requisites to provide the right RBAC permissions to the serviceaccount running tiller. 
+
+#### Helm install errors with `Job failed: Deadline exceeded`
+
 ```
+helm install --debug --set dataInterface=eth1,managementInterface=eth1,etcdEndPoint=etcd:http://192.168.20.290:2379,clusterName=$(uuid) ./charts/px/
+[debug] Created tunnel using local port: '39771'
 
+[debug] SERVER: "127.0.0.1:39771"
+
+[debug] Original chart version: ""
+[debug] CHART PATH: /root/k8s-1.7/helm/charts/px
+
+Error: Job failed: DeadlineExceeded
+```
+This error indicates that the pre-install hook for the helm chart has failed to run to completion correctly. Verify that the etcd URL is accessible. This error is also visible on kubernetes cluster with version below 1.8 
```

Comment:
> This error is also visible on kubernetes cluster with version below 1.8 

Why have this special line for versions < 1.8? is it because there are multiple failed job pods instead of one? Apart from that distinction, this troubleshooting section applies equally to all k8s versions. So remove this line.

Reply:
@harsh-px so with < v1.8 the `backoffLimit` attribute doesnt exist, Hence the attribute to use is to set a `deadlinelimit` in seconds. Otherwise the job would keep spawning pods forever. 

This error is ONLY visible on < 1.8 since post that we use the `backoffLimit` attibute which denotes the number of retries.